src/pages/home/base/mobile-core/lifecycle/index.md

```diff
@@ -47,7 +47,7 @@ Using the `MobileCore.initialize` API to initialize the Adobe Experience Platfor
 
 <InlineAlert variant="info" slots="text"/>
 
-Lifecycle tracking is enabled by default when the `MobileCore.initialize` API is used and lifecycle extension is included as an app dependency. The following instructions only apply when using [manual extension registration](/src/pages/home/getting-started/get-the-sdk/#b-manual-extension-registration-using-mobilecoreregisterextensions-api) to register Lifecycle extension.
+Lifecycle tracking is enabled by default when the `MobileCore.initialize` API is used and Lifecycle extension is included as an app dependency. The following instructions only apply if [lifecycleAutomaticTrackingEnabled](https://developer.adobe.com/client-sdks/home/base/mobile-core/api-reference/#initoptions) is false or when using [manual extension registration](/src/pages/home/getting-started/get-the-sdk/#b-manual-extension-registration-using-mobilecoreregisterextensions-api) to register Lifecycle extension.
```

Comment:
Can you change the link to use `/src/pages/home/base/mobile-core/api-reference/#initoptions`?